test/Bridges/lazy_bridge_optimizer.jl

```diff
@@ -610,7 +629,19 @@ Objective function of type `MOI.ScalarQuadraticFunction{$T}` is not supported an
   (5) `MOI.ScalarQuadraticFunction{$T}`-in-`MOI.LessThan{$T}` constraints are not supported
 """
         MOIB.add_bridge(bridged, MOIB.Constraint.ScalarizeBridge{T})
-        @test debug_string(MOIB.debug_supports, attr) == "Objective function of type `MOI.ScalarQuadraticFunction{$T}` is supported.\n"
+        @test debug_string(MOIB.debug_supports, attr) == "Objective function of type `MOI.ScalarQuadraticFunction{$T}` is supported. To see bridging strategy use print_supporting_scheme.\n"
+        @test debug_string(MOIB.print_supporting_scheme, attr) == """
+|1| objective function of type `MOI.ScalarQuadraticFunction{$T}` is bridged (distance 12) by MOIB.Objective.SlackBridge{$T,MOI.ScalarQuadraticFunction{$T},MOI.ScalarQuadraticFunction{$T}} to
+  (1) `MOI.ScalarQuadraticFunction{$T}`-in-`MOI.GreaterThan{$T}` constraints are bridged (distance 5) by MOIB.Constraint.QuadtoSOCBridge{$T} to
+    (2) `MOI.VectorAffineFunction{$T}`-in-`MOI.RotatedSecondOrderCone` constraints are bridged (distance 4) by MOIB.Constraint.VectorSlackBridge{$T,MOI.VectorAffineFunction{$T},MOI.RotatedSecondOrderCone} to
```

Comment:
Here this block is printed twice. To avoid this, we should have something like
```
|1| ...
   |*| objective function for type `MOI.SingleVariable` is natively supported
   (1) ... constraints are bridged
(1) ... constraints are bridged (distance 5) by ...
```
here we use a star `|*|` and not a number because since this node is not natively supported, we won't detail how it's bridge.
In `(1) ... constraints are bridged` we just say it's bridged, no other details, the reader is refered to `(1)` for more details.